processing/app/models.py

```diff
@@ -1,30 +1,94 @@
 import uuid
-from datetime import datetime, timezone
-from pydantic import BaseModel, UUID4
-from sqlmodel import Field, SQLModel
+from datetime import datetime
 from enum import StrEnum
+from typing import Optional
 
-class Item(BaseModel):
-    name: str
-    price: float
-    is_offer: bool = False
+from pydantic import BaseModel
+from pydantic import UUID4
+from sqlmodel import Column
+from sqlmodel import Field
+from sqlmodel import Relationship
+from sqlmodel import SQLModel
+from sqlmodel import String
 
-class StatusEnum(StrEnum):
+
+class TaskStatus(StrEnum):
     NULL = "null"
-    PENDING = "pending"
+    IN_PROGRESS = "in_progress"
     COMPLETED = "completed"
     ERROR = "error"
 
-class AnalysisJob(SQLModel, table=True):
+
+# separate Task with answer into separate model? (AnalysisTaskCompleted with result:str + status:COMPLETED)
+class Task(SQLModel, table=True):
     id: UUID4 = Field(default_factory=uuid.uuid4, primary_key=True)
-    survey_id: str = Field(index=True)
-    created_at: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
-    status: StatusEnum = Field(default=StatusEnum.NULL)
+    survey_id: UUID4 = Field(index=True)
+    # TODO: parametrize default_factory depending on database engine (sync/async),
+    # since async engine use only offset-naive datetime
+    created_at: datetime = Field(default_factory=lambda: datetime.now())
+    status: TaskStatus = Field(default=TaskStatus.NULL)
+    result: Optional[str] | None = Field(default=None)
+
+
+class Survey(SQLModel, table=True):
+    __tablename__ = "survey"  # type: ignore
+    id: UUID4 = Field(default_factory=uuid.uuid4, primary_key=True)
+    name: str = Field()
+    schemaJson: str = Field(sa_column=Column("schema_json", String))
+    answers: list["SurveyAnswer"] = Relationship(back_populates="survey")
+
+
+class SurveyAnswer(SQLModel, table=True):
+    __tablename__ = "survey_answers"  # type: ignore
+    id: UUID4 = Field(default_factory=uuid.uuid4, primary_key=True)
+    answersJson: str = Field(sa_column=Column("answers_json", String))
+
+    survey_id: UUID4 = Field(foreign_key="survey.id")
+    survey: Optional["Survey"] = Relationship(back_populates="answers")
+
+
+class AnalysisJobResponse(BaseModel):
```

Comment:
Move non-SQLModel models to `app/schemas.py` to differentiate between application related models and database models.